### ISSUE TITLE
Code fix suggests to import containing namespace (CS1503)

### ISSUE THREAD

**Issue by JosefPihrt** [dotnet/roslyn#13147]
**Description**:
Argument in ```WithModifiers```  invocation reports CS1503. Problem is that code fix to this error suggests to import namespace ```Foo``` which is containing namespace.

**Code**:
```csharp
using System.Collections.Generic;
using System.Linq;
using Microsoft.CodeAnalysis.CSharp;
using Microsoft.CodeAnalysis.CSharp.Syntax;
using static Microsoft.CodeAnalysis.CSharp.SyntaxFactory;

namespace Foo
{
    public class Bar
    {
        public PropertyDeclarationSyntax CreatePropertyDeclaration()
        {
            var kinds = new List<SyntaxKind>();
            kinds.Add(SyntaxKind.PublicKeyword);

            return PropertyDeclaration(default(TypeSyntax), "identifier")
                .WithModifiers(kinds); // CS1503 is reported here
        }
    }

    public static class PropertyDeclarationExtensions
    {
        public static PropertyDeclarationSyntax WithModifiers(
            this PropertyDeclarationSyntax propertyDeclaration,
            params SyntaxKind[] kinds)
        {
            return propertyDeclaration.WithModifiers(TokenList(kinds.Select(token => Token(token))));
        }
    }
}
```

**Comment by svick** [comment]
This sounds like something that might have been fixed by #13011.

**Comment by Pilchie** [comment]
@kuhlenh and I thought this looked familiar, but couldn't find a duplicate.

**Comment by CyrusNajmabadi** [comment]
Fixed with db527077c3a29371f309fcd69e90897ea6568f5d
